pkg/kubelet/metrics/metrics.go

```diff
@@ -412,12 +412,13 @@ var (
 		[]string{"runtime_handler"},
 	)
 
-	// RunningPodCount is a gauge that tracks the number of Pods currently running
+	// RunningPodCount is a gauge that tracks the number of Pods currently  in running phase
+	// It is used to expose the kubelet internal state: how many pods have running containers in the container runtime, and mainly for debugging purpose.
 	RunningPodCount = metrics.NewGauge(
 		&metrics.GaugeOpts{
 			Subsystem:      KubeletSubsystem,
 			Name:           RunningPodsKey,
-			Help:           "Number of pods currently running",
+			Help:           "Number of pods currently in running phase",
```

Comment:
IIUC, running_pods == pause container number >= pods with pause container number >= pods in running phase.

So "in running phase" doesn't seem to be accurate here.

Reply:
so running-pod number here is actually pause-container number？

Reply:
Updated